.travis.yml

```diff
@@ -52,21 +48,15 @@ matrix:
       python: 3.5
     - env: TOX_SUFFIX="requests1"
       python: 3.6
-    - env: TOX_SUFFIX="aiohttp"
-      python: 2.6
     - env: TOX_SUFFIX="aiohttp"
       python: 2.7
     - env: TOX_SUFFIX="aiohttp"
       python: pypy
-    - env: TOX_SUFFIX="aiohttp"
-      python: pypy3
 python:
-- 2.6
 - 2.7
 - 3.5
 - 3.6
 - pypy
-- pypy3
```

Comment:
any chance of adding:
```yaml
  - "pypy2.7-5.8.0"
  - "pypy3.5-5.9.0"
```

Reply:
eg instead of removing `pypy3` replace it with `pypy3.5-5.9.0`

Reply:
Added.